README.md

```diff
@@ -136,12 +137,30 @@ Or you can import the CSS:
 
 Here's how you can get started developing locally:
 
+1. Clone this repo and create a symlink:
```

Comment:
I updated this section with the steps I did to run it locally

Reply:
`create a symlink` is confusing since it may give the impression the user needs to symlink by hand.
Maybe: "and link it to your global node_modules"?

Reply:
👍